pages/learner-onboarding/onboarding-done.js

```diff
@@ -1,6 +1,16 @@
 import LearnerOnboardingLayout from "../../components/LearnerOnboardingLayout";
+import { useEffect } from "react";
+
+export default function OnboardingDone({
+  setOnboarding,
+  background,
+  avatarNameInLocalStorage,
+}) {
+  useEffect(() => {
+    document.body.style.backgroundColor = background;
+    setOnboarding(true);
+  }, []);
```

Comment:
The ``setOnboarding`` thing can remain in a ``useEffect`` here, but ditto above re the background colour thing.